sentry-android-core/src/main/java/io/sentry/android/core/EnvelopeFileObserverIntegration.java

```diff
@@ -17,6 +17,7 @@
   private @Nullable EnvelopeFileObserver observer;
   private @Nullable ILogger logger;
 
+  @SuppressWarnings("SyntheticAccessor")
```

Comment:
there are a bunch of SyntheticAccessor errors for private static final classes, not sure if they make sense or are false positive, I don't see how could it be problematic